test-app/overlays/dev/platform-deploy.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: platform
+spec:
+  replicas: 1
+  template:
+    metadata:
+      annotations:
+        reloader.stakater.com/search: "true"
+    spec:
+      initContainers:
+        - name: init-platform
+          image: docker.pkg.github.com/virtocommerce/vc-deploy-containers/platform-installer:dev
+          command:
+            - "/bin/sh"
+            - "-c"
+          args:
+            - "echo START INIT $(date +%Y-%m-%d-%H-%M-%S) 
+            && python3 install-modules.py $(VC_PLATFORM_MODULES_CONFIG) /mnt/Modules/
+            && rm -rf $(ELECTRONICS_THEME_LOCATION)/* && /install-theme.sh $(ELECTRONICS_THEME_URL) $(ELECTRONICS_THEME_LOCATION)
+            && echo ELECTRONICS THEME INSTALLED $(date +%Y-%m-%d-%H-%M-%S)
+            && rm -rf $(B2B_THEME_LOCATION)/* && /install-theme.sh $(B2B_THEME_URL) $(B2B_THEME_LOCATION)
+            && echo B2B THEME INSTALLED $(date +%Y-%m-%d-%H-%M-%S)
+            && python3 upload-assets.py $(ASSETS_URL) $(ASSETS_LOCATION)
+            && echo ASSETS INSTALLED $(date +%Y-%m-%d-%H-%M-%S)"
+          envFrom:
+            - configMapRef:
+                name: cms-content-cm
+          env:
+            - name: VC_PLATFORM_MODULES_CONFIG
+              value: https://raw.githubusercontent.com/VirtoCommerce/vc-deploy-apps/master/test-app/overlays/dev/modules.json   
+          volumeMounts:
+            - mountPath: /mnt/Modules
+              name: modules-data
+            - mountPath: /mnt/cms-content
+              name: cms-content-data
+      containers:
+        - name: vc-platform-web   
+          env:
+            - name: REDIS_PASS
+              valueFrom:
+                secretKeyRef:
+                  name: redis
+                  key: REDIS_PASS
+            - name: APPINSIGHTS_INSTRUMENTATIONKEY
```

Comment:
Please comment out APPINSIGHTS_INSTRUMENTATIONKEY variable and its value to prevent AppInsight unauthorized use